### PR TITLE
Part5

### PR DESCRIPTION
**Pull request by kstoops815**

### CODE REVIEW

**Comment by kstoops815** [comment]
I added an "isSuper" key to the contacts. I added a "superpowers" button from Font Awesome. When clicked, the isSuper value is changed to true and the background changes to a Justice League symbols image. When clicked again, the background is removed and "isSuper" is set to false.

File: javascripts/controllers/FavoritesCtrl.js [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 app.controller("FavoritesCtrl", function($location, $rootScope, $scope, ContactsService){
 
+	$scope.$on('$viewContentLoaded', function(event){
+    console.log('content loaded!');
```
Comment (zoeames) [inline javascripts/controllers/FavoritesCtrl.js]: remove this console

File: javascripts/controllers/ViewCtrl.js [hunk 2]
```diff
@@ -3,6 +3,10 @@
 app.controller("ViewCtrl", function($location, $rootScope, $scope, ContactsService){
 	$scope.contact = {};
 
+	$scope.$on('$viewContentLoaded', function(event){
```
Comment (zoeames) [inline javascripts/controllers/ViewCtrl.js]: do you need this $on?

File: sass/main.scss [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+	background-color: $red;
+}
+
+.login {
+	height: 75px;
+	width: 400px;
+}
+
+
+
```
Comment (zoeames) [inline sass/main.scss]: delete extra blank lines

**Comment by zoeames** [comment]
@kstoops815 All functionality is in.  Please go back through and clean up the code - no console.logs (except in .catch).  Remove extra spaces, remove functions you don't need

**Comment by kstoops815** [comment]
Code cleanup done.

**Comment by zoeames** [comment]
yay looks good! 👍

**Pull request merged**